### PR TITLE
Keyvault Integration

### PR DESCRIPTION
**Pull request by sushilraje**
We are integrating key vault in RM. Hence, in RM deployments we would not have auth required and tenant id as environment variables. Hence, a script has been added to call keyvault and set the necessary variables before in the EntryPoint of Docker container. The script is benign, as in, if Keyvault name, AppId, and AppSecret are available through environment variables, only then the script would execute.
# Type of change? 

- [ ] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 

...

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

---
This change is [Reviewable](https://reviewable.io/reviews/azure/pcs-remote-monitoring-webui/1234)

### CODE REVIEW

File: scripts/docker/content/set_env.sh [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  _temp=`echo $1 | sed 's/\\\\\//\//g' | sed 's/[{}]//g' | awk -v k="text" '{n=split($0,a,","); for (i=1; i<=n; i++) print a[i]}' | sed 's/\"\:\"/\|/g' | sed 's/[\,]/ /g' | sed 's/\"//g' | grep -w $2`;
+  echo ${_temp##*|};
+}
+
+############# Main function #############
+
+modify_webui_config() {
+
+  if [ "$1" == "AUTH" ]; then
+    sed -i "s/authEnabled.*/authEnabled: '$2',/g" C:\\Users\\suraje\\PCS\\work\\pcs-remote-monitoring-webui\\public\\webui-config.js
```
Comment (ppathan) [inline scripts/docker/content/set_env.sh]: You probably want to remove the hardcoded path.

File: scripts/docker/content/set_env.sh [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  fi
+
+  if [ "$1" == "TENANT" ]; then
+    sed -i "s/tenant.*/tenant: '$2',/g" C:\\Users\\suraje\\PCS\\work\\pcs-remote-monitoring-webui\\public\\webui-config.js
```
Comment (ppathan) [inline scripts/docker/content/set_env.sh]: Path is hardcoded

File: scripts/docker/content/set_env.sh [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  fi
+
+  if [ "$1" == "INSTANCE_URL" ]; then
+    if [ "$2" == "" ]; then
+
+      sed -i "s/instance.*/instance: 'https\:\/\/login\.microsoftonline\.com\/'/g" C:\\Users\\suraje\\PCS\\work\\pcs-remote-monitoring-webui\\public\\webui-config.js
```
Comment (ppathan) [inline scripts/docker/content/set_env.sh]: Path is hardcoded

File: scripts/docker/content/set_env.sh [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    else
+      sed -i "s/instance.*/instance: '$2'/g" C:\\Users\\suraje\\PCS\\work\\pcs-remote-monitoring-webui\\public\\webui-config.js
```
Comment (ppathan) [inline scripts/docker/content/set_env.sh]: Path is hardcoded

**Comment by ppathan** [comment]
@isaac-dasan Can you also take a look?

File: src/components/pages/maintenance/maintenance.js [hunk 5]
```diff
@@ -55,9 +55,9 @@ export class Maintenance extends Component {
 
   getData = (deviceEntities, timeInterval) => {
     const deviceIds = Object.keys(deviceEntities || this.props.deviceEntities);
-    const devices = deviceIds.length ? deviceIds.join(',') : undefined;
+    const Devices = deviceIds.length ? deviceIds.join(',') : undefined;
```
Comment (isaac-dasan) [inline src/components/pages/maintenance/maintenance.js]: camelCased variables are a convention. We cannot break that here.

File: src/services/telemetryService.js [hunk 6]
```diff
@@ -92,21 +92,21 @@ export class TelemetryService {
...
+  static getTelemetryByDeviceIdP1M(Devices = []) {
     return TelemetryService.getTelemetryByMessages({
-      from: 'NOW-PT1M',
-      to: 'NOW',
-      order: 'desc',
-      devices
+      From: 'NOW-PT1M',
+      To: 'NOW',
+      Order: 'desc',
+      Devices
```
Comment (isaac-dasan) [inline src/services/telemetryService.js]: If you want the API to receive 'Devices', just do this -> `Devices: devices`. You don't have to change the variable name everywhere.

1. These methods been working well already correct? 
2. why this change now? 
3. How is this related to KeyVault stuff? 

If this is unrelated I'd recommend creating a separate PR.
